docs/specifications/broker-authentication.md

```diff
@@ -0,0 +1,107 @@
+# Broker Authentication
+
+This document attempts to outline the high-level plan for adding authentication
+to the broker api routes.
+
+When talking to service brokers, the [Service Catalog uses basic auth](#service-catalog-basic-auth). The Ansible Service Broker will implement Basic Auth using a secret shared between the catalog and the broker. The authentication will be configurable allowing a user i.e. developer to disable it.
+
+The plan for this sprint is to do the following:
+
+* make auth routes configurable by enhancing the configuration
+```
+auth:
+   - type: basic | INSERT OTHER TYPE HERE
+   - enable: true | false
```

Comment:
wondering if this should be part of the broker section or its own section?

Its own section.

```yaml
broker:                    
  dev_broker: true         
  launch_apb_on_bind: false
  recovery: true           
  output_request: true     
auth:                      
  type: basic              
  enable: true             
```

Part of broker.
```yaml
broker:                    
  dev_broker: true         
  launch_apb_on_bind: false
  recovery: true           
  output_request: true     
  auth:                      
    type: basic              
    enable: true             
```

Opinions?